setup.py

```diff
@@ -113,7 +113,7 @@
               'wcsaxes',
           ],
           test=[
-              'pytest-mpl',
```

Comment:
I'd prefer we leave `pytest-mpl` here in `setup.py`.
We plan to use it very soon, so we added it to `setup.py` and `.travis.yml` already.